home.php

```diff
@@ -16,31 +16,13 @@
 		<main id="main" class="site-main">
 
       <?php
-      if ( ! is_front_page() ) :
-        ?>
-
-        <header class="page-header">
-          <div class="wrap">
-
-            <h1 class="page-title"><?php single_post_title(); ?></h1>
-
-          </div><!-- .wrap -->
-        </header><!-- .page-header -->
-
-        <?php
-        if( has_post_thumbnail( get_option( 'page_for_posts', true ) ) ) :
-          ?>
-
-          <div class="page-thumbnail">
-            <?php echo get_the_post_thumbnail( get_option( 'page_for_posts', true ) ); ?>
-          </div><!-- .post-thumbnail .page-thumbnail -->
-
-          <?php
-        endif;
-        ?>
```

Comment:
@eric-michel We need to test this. In scenarios where the `posts` page is not the front page it looks like we run the risk of not having a `<header class="page-header">` element on those pages which I believe is a persistent element across all other templates (again, need to test to verify).

I worry that by removing it we'll introduce a template with a unique structure that could affect layout / styling.

Reply:
@dswebsme in looking at our normal page template, it doesn't look like a `<header class="page-header">` is ever generated. It's always `entry-header`, and only if the title is not hidden. This brings the main blog page in line with those templates, if I'm looking at things right.

Reply:
The traditional separation between `page` and `post` in templates presents itself in cases like this where `.page-header` and `.entry-header` are used respectively. Archive pages can get particularly interesting such that the overall template utilizes `.page-header` while individual posts rendered by the loop might have their own `.entry-header` associated with each post.

When I search the code for `page-header` I find about what I'd expect to see in `emma`:
<img width="320" alt="page-header" src="https://user-images.githubusercontent.com/9062957/104641769-329ac480-5678-11eb-9fc8-9caffc0cfe3e.png">

Reply:
^ Notice that `single.php` is not on this list as it utilizes `entry-header` in its' related template parts, namely `content.php`.

```<div class="entry-content">```

Reply:
Upon further investigation is looks like there is some inconsistency in how / where we use `page-header` vs. `entry-header` so I'm resolving this conversation so we can address that in a separate issue.